javagen/src/main/java/com/azure/autorest/model/clientmodel/ExternalDocumentation.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.model.clientmodel;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Objects;
+
+/**
+ *  External documentation at operation level, can be used to link to rest API documentation
```

Comment:
This external documentation doesn't have to be operation level (it can be at client level). So, the javadoc should be updated.